pages/services/kafka/2.6.0-2.2.1/release-notes/index.md

```diff
@@ -0,0 +1,151 @@
+## Version 2.5.0-2.1.0
+
+### Important Notes
+
+* The `inter_broker_protocol_version` now defaults to the newer, `2.1` version. This has a few implications, as described below:
+
+    - Kafka 1.1.0 supports `inter_broker_protocol_version`: `1.1` maximum, and by default it is set to `1.0`.
+    - Kafka 2.1.0 supports `inter_broker_protocol_version`s up to `2.1`. 
+    - If you haven't specified a `inter_broker_protocol_version` in your options file, the new default will be used and changed to `2.1`.
+
+    The problem with this is that it will cause downtime;  during the upgrade, some Kafka nodes will be on Kafka 1.1.0 using `inter_broker_protocol_version` `1.0` and others will be on Kafka 2.1.0 using protocol `2.1`.
+
+    To avoid any potential downtime, change the protocol version used when upgrading Kafka.
+
+    - Set up CLI to connect to a soak cluster
+    - Update your `options_file.json` with the following contents:
+        ```
+        {
+            ...
+            "kafka": {
+                ...
+                "inter_broker_protocol_version": "1.0"
+                ...
+            }
+            ...
+        }
+        ```
+
+    - And update your service like so:
+       ```
+       ~$ dcos package install --cli --yes kafka
+       ~$ dcos kafka --name=data-services/kafka update start \
+           --package-version=2.5.0-2.1.0 \
+           --options=options_file.json
+       ```
+
+### Updates
+- Upgrade {{ model.techShortName }} base tech to version 2.1.0. See [{{ model.techShortName }}'s Release Notes](https://kafka.apache.org/21/documentation.html#upgrade_210_notable) more for details.
+- Upgrade Zookeeper Client version to 3.4.13.
+
+## Version 2.4.0-1.1.1
+
+### Updates
+- Upgrade {{ model.techShortName }} base tech to version 1.1.1. See [{{ model.techShortName }}'s Release Notes](https://archive.apache.org/dist/kafka/1.1.1/RELEASE_NOTES.html) for details.
+- Upgrade the base dcos-commons SDK version to 0.55.2.
+- Upgrade Zookeeper Client version to 3.4.13.
+
+### New Features
+- Number of open file descriptors is now configurable via the `RLIMIT_NOFILE_SOFT` and `RLIMIT_NOFILE_HARD` configuration parameters.
+- Timeouts for readiness checks are now configurable via the `READINESS_CHECK_INTERVAL`, `READINESS_CHECK_DELAY` and `READINESS_CHECK_TIMEOUT` configuration parameters.
+
+## Version 2.3.0-1.1.0
+
+### Updates
+- Upgrade {{ model.techShortName }} base tech to version 1.1.0. See [{{ model.techShortName }}'s Release Notes](https://kafka.apache.org/11/documentation.html#upgrade_110_notable) for details.
+
+## Version 2.3.0-1.0.0
+
+### New Features
+- Support for configuring {{ model.techShortName }} transport encryption ciphers with secure defaults.
+
+## Version 2.2.0-1.0.0
+
+### New Features
+- Support for using a custom top level domain to facilitate exposing the service securely outside of the cluster. Details [here](/services/kafka/2.3.0-1.0.0/security/#securely-exposing-dcos-apache-kafka-outside-the-cluster).
```

Comment:
Rewrite `Support for using a custom top level domain to facilitate exposing the service securely outside of the cluster. Details [here](/services/kafka/2.3.0-1.0.0/security/#securely-exposing-dcos-apache-kafka-outside-the-cluster).` with `Support for using a custom top level domain to facilitate [exposing the service securely outside of the cluster](/services/kafka/2.3.0-1.0.0/security/#securely-exposing-dcos-apache-kafka-outside-the-cluster).